src/theme/CodeBlock/index.js

```diff
@@ -0,0 +1,270 @@
+/**
```

Comment:
do we need this file explicitly? I thought Docusaurus already supports Prism out-of-the-box, no? oh well, is this our customized implementation thing you're were mentioning in the PR comments? @lkrell

Reply:
double-checking with the instructions and our config in the repo, I think we already have this feature/plugin enabled.

see https://github.com/saucelabs/docusaurus-theme-github-codeblock
and also see our `docusaurus.config.js` at line 288

Reply:
We need this for swizzling. It's to modify css which is autogenerated each build. You swizzle and modify the css for the swizzle to have it work together. :)